srtcore/api.cpp

```diff
@@ -1512,7 +1528,7 @@ int CUDTUnited::epoll_release(const int eid)
 
 CUDTSocket* CUDTUnited::locateSocket(const SRTSOCKET u, ErrorHandling erh)
 {
-    CGuard cg(m_GlobControlLock);
+    CGuard cg (m_GlobControlLock);
 
```

Comment:
unintentional change?

Reply:
Changed besides. This change is intentional - the open brace sticks to the name only if the preceding name is a function (including type name as for the temporary object constructor call).